docs/en/latest/config.json

```diff
@@ -200,6 +200,31 @@
           "type": "doc",
           "id": "building-apisix"
         },
+        {
+          "type": "doc",
+          "id": "external-plugin"
+        },
+        {
+          "type": "doc",
+          "id": "plugin-develop"
+        },
+        {
+          "type": "doc",
+          "id": "wasm"
+        },
+        {
+          "type": "link",
+          "label": "CODE_STYLE",
```

Comment:
What's the order of the sections?

Reply:
Hi, I readjusted the sorting. Here, users need to understand the content related to APISIX development and then develop it at the end. Do you have a better suggestion? @spacewander

Reply:
It's OK. I am just being curious.